openssl-1.1.1/VITABUILD

```diff
@@ -4,6 +4,7 @@ pkgrel=1
 url="https://github.com/SonicMastr/vita-openssl3"
 source=("git+https://github.com/SonicMastr/vita-openssl3.git")
 sha256sums=('SKIP')
+depends=('libc')
```

Comment:
i got cpython3 patch.

can you explain this?
where is `libc`?

Reply:
Nowhere. It's a "fake" dependency to force openssl-1.1.1 into build_depends job

Reply:
I see. so if we use `pacman` correctly, this rule is broken rule that because we cannot find libc.
and also I got it on the slack message

workflow have to fix instead this way.

Reply:
**if** we use pacman correctly we
1) wouldn't have this problem
2) would need to adjust pkgbuilds anyway (to make use of makedepends)